### PR TITLE
Conveyor refactor attempt 2

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Pls don't let it stale again
https://github.com/tgstation/TerraGov-Marine-Corps/pull/4861

## Changelog
🆑
add: Conveyors can now be made in autolathes
refactor: refactored conveyors
/🆑

### CODE REVIEW

**Comment by TiviPlus** [comment]
@Hughgent

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -44,107 +89,117 @@
...
+		if(!(machine_stat & BROKEN))
+			verted = verted * -1
+			update_move_direction()
+			to_chat(user, "<span class='notice'>You reverse [src]'s direction.</span>")
+
+	else if(user.a_intent != INTENT_HARM)
+		user.transferItemToLoc(I, drop_location())
 	else
-		user.transferItemToLoc(I, loc)
+		return ..()
```
Comment (Hughgent) [inline code/modules/recycling/conveyor2.dm:189]: Ain't these supposed to be `screwdriver_act()` and what not?

Comment (TiviPlus) [reply code/modules/recycling/conveyor2.dm:189]: crap

**Review by Hughgent — APPROVED** [review]

**Comment by Hughgent** [comment]
no bug reports

**Pull request merged**